test/abandoned_checkouts_test.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AbandonedCheckoutsTest < Test::Unit::TestCase
+  test ":get lists all abandoned checkouts indexed by id" do
+    fake 'checkouts', method: :get, status: 200, body: load_fixture('checkouts')
+    abandoned_checkout = ShopifyAPI::AbandonedCheckout.all
+    assert_equal 450789469, abandoned_checkout.first.id
```

Comment:
Since we load a list we should assert that `abandoned_checkouts.size == load_fixture('checkouts').size`

also naming nitpick: `abandoned_checkout` should be pluralized since it is a list of abandoned checkouts